### PR TITLE
com.cassidyjames.plausible

### PR DESCRIPTION
**Pull request by cassidyjames**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by TheEvilSkeleton** [comment]
bot, build com.cassidyjames.plausible

File: com.cassidyjames.plausible.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+runtime-version: '3.38'
+base: io.elementary.BaseApp
+base-version: 'juno-20.08'
+sdk: org.gnome.Sdk
+command: com.cassidyjames.plausible
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=network
+  - --socket=session-bus
```
Comment (bilelmoussaoui) [inline com.cassidyjames.plausible.yaml:12]: This one is a bit too much, what do you need the session bus for ? You can run the app with flatpak run --log-session-bus to figure out which dbus name you need access to.

**Review by bilelmoussaoui — CHANGES_REQUESTED** [review]

**Comment by TheEvilSkeleton** [comment]
Didn't expect the issue to actually get closed... Sorry :sweat_smile:

File: com.cassidyjames.plausible.yaml (hunk 1 above)
Comment (TheEvilSkeleton) [reply com.cassidyjames.plausible.yaml:12]: Weird. It used to require session bus before, otherwise it wouldn't launch at all.

Comment (TheEvilSkeleton) [inline com.cassidyjames.plausible.yaml:12]:
```suggestion
```

**Comment by barthalion** [comment]
bot, build com.cassidyjames.plausible

**Pull request closed without merging**